Tasks/UsePythonVersion/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "use-python-version",
```

Comment:
We should change the task name to "UsePythonVersion" (pascal-cased) so that it's more [consistent with other task names](https://mseng.visualstudio.com/DefaultCollection/VSOnline/_wiki/wikis/VSOnline.wiki?wikiVersion=GBwikiMaster&pagePath=%2FCIX%2FTask-YAML) in YAML:
```yaml
steps:
- task: UsePythonVersion@0
  inputs:
    versionSpec: '1.8'
    outputVariable: yourMama
    addToPath: true
```

Reply:
This is the name of the NPM package for this task.  The task name is in task.json -- I believe that is what will be used in YAML.

Reply:
You're right, I mixed up the files.  Thanks.